fuzz/build.info

```diff
@@ -29,8 +29,9 @@ IF[{- !$disabled{"fuzz-afl"} || !$disabled{"fuzz-libfuzzer"} -}]
     PROGRAMS{noinst}=x509
   ENDIF
 
-  IF[{- !$disabled{"dtls"} -}]
+  IF[{- !$disabled{"sock"} -}]
     PROGRAMS{noinst}=dtlsclient dtlsserver
+    PROGRAMS{noinst}=dtlsclient-test dtlsserver-test
```

Comment:
You can disable dtls without disabling sockets, and in that case the dtls tests should not run.

Reply:
yes, i got confused. fixed.